002-routing-params-views/src/app/root.component.js

```diff
@@ -0,0 +1,11 @@
+(function(){
+    'use strict';
+
+    var root = {
+        templateUrl: 'app/root.html'
+    };
```

Comment:
if this is not going to be use elsewhere put it directly in the component declaration, this add unnecessary complexity to the component declaration